client/src/api/treeTrackerApi.js

```diff
@@ -1,6 +1,15 @@
 import { handleResponse, handleError } from "./apiUtils";
 import {session} from "../models/auth";
 
+
+function getOrganization(){
```

Comment:
Looks like we are replicating this code block ( I also see it above).  Can we break this out into a function call in a shared file so that we don't make any mistake.